README.md

```diff
@@ -1,31 +1,30 @@
-## Prerequisites
-PHP5.6 and MYSQL
-## Setup instructions
-Create a mysql database for the forums & create a user with full privileges to that database:  
-``mysql``  
-``> CREATE DATABASE {db_name}``  
-``> CREATE USER '{db_user}'@'localhost' IDENTIFIED BY '{db_password}';``  
-``> GRANT ALL PRIVILEGES ON {db_name}.* TO '{db_user}'@'localhost';``  
+### Important:
+Default admin account is ``Wetfish Online`` and the password is ``changeme``. **change it.**
 
-Go to the base directory of this repo:  
-``cd /var/www/html/online`` 
-Import the database schema:  
-``mysql -u root -p {db_name} < setup/schema.sql``  
+## How do I deploy this container stack?
 
-## Running with docker
+See [https://github.com/wetfish/production-manifests](https://github.com/wetfish/production-manifests)
+for production deployment and full stack dev env info.
 
-1. Create a **.env** based on the `.env.example` provided
-2. Run `docker-compose up -d`
+For development, to run just this stack, do 
+```bash
+cp mariadb.env.example mariadb.env
+# -> edit, change passwords and other info as needed
+cp php.env.example php.env
+# -> edit, change passwords to match
 
-## Repair database settings:
-First, in the ``forum`` directory of this repo, copy Settings.example.php to Settings.php  
-``cp forum/Settings.example.php forum/Settings.php``
+docker compose \
+  -f docker-compose.dev.yml \
+  up -d \
+  --build \
+  --force-recreate
 
-Next, Download the repair_settings tool from SMF: https://wiki.simplemachines.org/smf/Repair_settings.php
+docker compose -f docker-compose.dev.yml logs -f
+```
 
-Place repair_settings.php in the ``forum`` directory and visit the url, eg: ``127.0.0.1/online/forum/repair_settings.php``  
-Fill in the MySQL Database Info section and save.
-Reload the page and click **[Restore all settings]** at the bottom, save.
+The service will be available at [http://127.0.0.1:2404](http://127.0.0.1:2404)
```

Comment:
This should actually be `http://127.0.0.1:2404/forum/`, otherwise you get redirected

Reply:
Addressed in [1a941ad](https://github.com/wetfish/online/pull/135/commits/1a941adc3fa6b597bf0f4daa0ed4e0005fcc6c26)